docs/source/faq.rst

```diff
@@ -0,0 +1,29 @@
+.. include:: ./links.inc
+.. _faq:
+
+FAQ
+===
+
+Q1: Why do my file names look so weird?
+---------------------------------------
+
+This is due to restriction that (Dataverse_) currently puts onto file names and directory names.
+
+Let us explain that in a little more detail. Dataverse allows only directory names that have characters from
+the english alphabet, numbers, and the characters " ", "-", "_", ".". There are similar restrictions for file names.
+That means, Dataverse cannot store a file name like "Änderungen" or "Déchiffrer", due to the "Ä" and "é" in them.
+Generally, Dataverse does not support names that contain letters from non-English alphabets.
```

Comment:
I feel like people could be confused because they indeed can upload files with weird names from the webinterface (there only directory restrictions apply)

![image](https://user-images.githubusercontent.com/29738718/225907302-fc2222e3-97df-4806-94c9-0d155d6019e9.png)

Maybe worthwhile stating why directory name restrictions relate to file name restrictions?
maybe also worthwhile to link to the dataverse API docs that specify this?